client/src/App.jsx

```diff
@@ -1,59 +1,49 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import {Icon}from 'antd';
-// import Customers from './Components/Layouts/Customers/index';
+import Customers from './Components/Layouts/Customers/index';
 import Sidebar from "./Components/CommonComponent/Sidebar";
-import Header from "./Components/CommonComponent/Header";
 import Navbar from "./Components/CommonComponent/Navbar";
-import Profile from "./Components/Layouts/SingleCustomer";
-
 import Login from "./Components/Layouts/Login";
 import Home from "./Components/Layouts/Home";
+import OrdersManagement from "./Components/Layouts/Ordersmanagement";
+import SingleCaptain from "./Components/Layouts/SingleCaptains";
+import SingleCastomer from "./Components/Layouts/SingleCustomer";
 import Captains from "./Components/Layouts/Captains"
 
 import "./App.css";
 
-import Viewcaptain from "./Components/Layouts/SingleCaptains";
-// import Captains from "./Components/Layouts/Captains/index"
-
 class App extends Component {
   state = {};
   render() {
     return (
       <div className="App">
         <Router>
+          <Route exact path="/login" component={Login} exact />
```

Comment:
@naremanhilles Minor thing: it looks like you have `exact` written twice

Reply:
@naremanhilles One more thing I noticed - since this route is outside of the `<Switch>`, it's possible that it could show AND the result of the switch could show, in the future.  Since all of your switch routes have "exact", it can't happen right now, but if in the future you had a `<Route component={NotFound} />` or something like that at the bottom of your switch, it could show up when the user is on /login.  Just something to watch for in the future 👀